docs/tutorials/Remark.adoc

```diff
@@ -0,0 +1,396 @@
+}
+----
+
+NOTE: Don't forget to update `AddressBookParser` to use our new `RemarkCommandParser`!
+
+If you are stuck, check out the sample link:https://github.com/j-lum/addressbook-level35/commit/c0f8419be4fb59bffafcd73026769459e5cf5c5e#diff-fc19ecee89c3732a62fbc8c840250508[here].
+
+== Add `Remark` to the model
+
+Now that we have all the information that we need, let's lay the groundwork for some _persistent_ changes.
+We achieve that by working with the `Person` model.
+Each field in a Person is implemented as a separate class (e.g. a `Name` object represents the person's name).
+That means we should add a `Remark` class so that we can use a `Remark` object to represent a remark given to a person.
+
+=== Add a new `Remark` class
+
+Create a new `Remark` in `seedu.address.model.person`. Since a `Remark` is a field that is similar to `Address`, we can reuse a significant bit of code.
+
+A quick copy-paste and search-replace later, you should have something like link:https://github.com/j-lum/addressbook-level35/commit/c8e6048f0d81f2b4b7797e838226da823892129b#diff-af2f075d24dfcd333876f0fbce321f25[this].
+
+=== Make use of `Remark`
+
+Let's change `RemarkCommand` and `RemarkCommandParser` to use the new `Remark` class instead of plain `String`.
+These should be relatively simple changes.
+
+== Add a placeholder element for remark to the UI
+
+Without getting too deep into `fxml`, let's go on a 5 minute adventure to get some placeholder text to show up for each person.
+
+Simply add
+[source, java]
+PersonCard.java
+```
+@FXML
+private Label remark;
+```
+
+to link:https://github.com/j-lum/addressbook-level35/commit/06f9897d7088d22689de478c94612f8bcbff2964#diff-0c6b6abcfac8c205e075294f25e851fe[`seedu.address.ui.PersonCard`].
+`@FXML` is an annotation that marks a private or protected field and makes it accessible to FXML.
+It might sound like Greek to you right now, don't worry -- we will get back to it later.
+
+Then insert
+
+```
+<Label fx:id="remark" styleClass="cell_small_label" text="\$remark" />
+```
+into link:https://github.com/j-lum/addressbook-level35/commit/06f9897d7088d22689de478c94612f8bcbff2964#diff-12580431f55d7880578aa4c16f249e71[`main/resources/view/PersonListCard.fxml`].
+
+That's it! Fire up the application again and you should see something like this:
+
+.$remark shows up in each entry
+image::$Remark.png[]
+
+== Modify `Person` to support a `Remark` field
+
+Since `PersonCard` displays data from a `Person`, we need to update `Person` to get our `Remark` displayed!
+
+=== Modify `Person`
+
+We change the constructor of `Person` to take a `Remark`.
+We will also need to define new fields and accessors accordingly to store our new addition.
+
+=== Update other usages of `Person`
+
+Unfortunately, a change to `Person` will cause other commands to break, you will have to modify these commands to use the updated `Person`!
+
+TIP: Use the `Find Usages` feature in IntelliJ IDEA on the `Person` class to find these commands.
```

Comment:
Tip only makes me overwhelmed seeing all these

![image](https://user-images.githubusercontent.com/27397021/64483997-89032200-d23e-11e9-8018-a177bf099efe.png)

What helped me was to run the app and see what broke during compilation 😆

Reply:
Technically they should have finished the removing address tutorial before this and have some competency navigating this. I have my fingers crossed.